compatibility_lib/compatibility_lib/configs.py

```diff
@@ -95,6 +104,29 @@ def _format_url(repo_name, setuppy_path=''):
     'grpc-google-iam-v1',
     'grpcio',
     'opencensus',
+    'opencensus-correlation',
```

Comment:
That's a good point - is the bottle neck just in the cloning or does the actual testing resource heavy as well?

Reply:
The testing resource (I assume you mean the github repo?) will grow heavier over time, as github will by default cloning all the revisions for that repo. And that will multiply by 24 as we have 24 packages living under that same opencensus repo. I think it's not that heavy for now as the opencensus extensions packages are created recently, but they will grow heavy ultimately with a lot revisions.

Reply:
It will make sense to run the loadtest by deploying this to our cloud project for test before actually deploying to prod server. The `get_compatibility_data.py`  script which the kokoro jobs runs can be run locally for recording the total time it takes for complete a run.

Reply:
I will do a compat lib release before merging this PR so that it will be easier to roll back any resource issues.